go/engine/paperkey.go

```diff
@@ -106,24 +106,40 @@ func (e *PaperKey) Run(ctx *Context) error {
 		}
 	}
 
-	ska := libkb.SecretKeyArg{
+	ska1 := libkb.SecretKeyArg{
 		Me:      me,
 		KeyType: libkb.DeviceSigningKeyType,
 	}
-	signingKey, err := e.G().Keyrings.GetSecretKeyWithPrompt(ctx.SecretKeyPromptArg(ska, "You must sign your new paper key"))
+	signingKey, err := e.G().Keyrings.GetSecretKeyWithPrompt(ctx.SecretKeyPromptArg(ska1, "You must sign your new paper key"))
 	if err != nil {
 		return err
 	}
 
+	ska2 := libkb.SecretKeyArg{
+		Me:      me,
+		KeyType: libkb.DeviceEncryptionKeyType,
+	}
+	encryptionKeyGeneric, err := e.G().Keyrings.GetSecretKeyWithPrompt(ctx.SecretKeyPromptArg(ska2, "You must encrypted for your new paper key"))
```

Comment:
Prompt language does not parse in the language of English, as derived from the old German and French.